REQUIREMENTS.md

```diff
@@ -2,7 +2,7 @@
 
 The dependencies listed here have varying installation instructions depending upon your preferred operating system. Please use in the installation instructions provided in [INSTALL.md](INSTALL.md).
 
-- `Chapel 1.26.0 or later`
+- `Chapel 1.27.0 or later`
```

Comment:
Are we not supporting 1.26.0 at all any further? I would update this to 

`Chapel 1.26.0 or later` (1.27.0 Recommended)

Reply:
I completely agree with what you are saying here. I would prefer that everyone stick with the latest version, but I am not sure that is truly going to be a reality. I am fine with keeping it at `1.27.0`, but then I would remove the `or greater`. My thinking here is that we don't have a higher version and we only want them using `1.27.0`. 

This is a good point though, we are really only testing the current `Chapel` version. My thought would then be to say we only actively support that version. I don't think we can eliminate "support" for older versions though because there is often a delay in customers updating.